policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_dependent_exemption.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class mt_dependent_exemption(Variable):
```

Comment:
what if there are more than 3? tax form gives special instructions

Reply:
According to [Montana legal code](https://www.leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0140/0150-0300-0210-0140.html), thereis no specific instruction for case of "3 or more dependents".

Reply:
The tax form says to check the instructions. Please do so.

Reply:
I've checked the instructions and believe that the instruction only suggests to manually allocate dependents to spouse in SEPARATE case.